post/stats.F90

```diff
@@ -1053,8 +1053,16 @@ end subroutine init_levs
                     zlev(lev).eq.750 .or. zlev(lev).eq.600 .or. &
                     zlev(lev).eq.500 .or. zlev(lev).eq.400 .or. &
                     zlev(lev).eq.300 .or. zlev(lev).eq.200 .or. &
-                    zlev(lev).eq.100 .or. zlev(lev).eq.70  .or. &
-                    zlev(lev).eq.30  .or. zlev(lev).eq.10  ) then
+                    zlev(lev).eq.100 .or. zlev(lev).eq.90  .or. &
```

Comment:
@wmputman Should we perhaps do something like:
```fortran
   integer, dimension(:), allocatable :: valid_heights
...
   valid_heights = &
      [ 925, 850, 750, 600, 500, 400, 300, 200, 100, 90, &
      80, 70, 60, 50, 40, 30, 20, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1]
...
   if (any(zlev(lev) == valid_heights)) then
...
```

or perhaps this is as many things as might ever be added, so might as well keep this?